*
* Visual Studio Code Server
*
* By using the software, you agree to
* the Visual Studio Code Server License Terms (https://aka.ms/vscode-server-license) and
* the Microsoft Privacy Statement (https://privacy.microsoft.com/en-US/privacystatement).
*
Server bound to 127.0.0.1:44921 (IPv4)
Extension host agent listening on 44921

[08:33:19] 




[08:33:20] Extension host agent started.
[08:33:20] [127.0.0.1][db7a493e][ExtensionHostConnection] New connection established.
[08:33:20] Deleted uninstalled extension from disk ms-python.debugpy /home/pi/.vscode-server/extensions/ms-python.debugpy-2024.2.0-linux-armhf
[08:33:20] Deleted uninstalled extension from disk ms-python.python /home/pi/.vscode-server/extensions/ms-python.python-2024.2.1
[08:33:20] [127.0.0.1][cddf52bf][ManagementConnection] New connection established.
[08:33:20] [127.0.0.1][db7a493e][ExtensionHostConnection] <861> Launched Extension Host Process.
New EH opened, aborting shutdown
[08:38:20] New EH opened, aborting shutdown
[08:38:34] [127.0.0.1][cddf52bf][ManagementConnection] The client has reconnected.
[08:38:34] [127.0.0.1][db7a493e][ExtensionHostConnection] The client has reconnected.
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at D.U (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:32471)
    at D.updateIcon (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:27683)
    at j.<computed> (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:22875)
    at Object.call (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:36:5137)
    at l.s (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:5145)
    at l.q (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:4647)
    at n.value (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at t.y (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:29:665)
    at t.fire (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:29:882)
    at process.ue (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:25:10030)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[08:38:35] Error [CodeExpectedError]: Could not find pty on pty host
    at D.U (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:32471)
    at D.updateIcon (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:27683)
    at j.<computed> (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:22875)
    at Object.call (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:36:5137)
    at l.s (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:5145)
    at l.q (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:4647)
    at n.value (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at t.y (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:29:665)
    at t.fire (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:29:882)
    at process.ue (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:25:10030)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at D.U (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:32471)
    at D.updateTitle (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:27632)
    at j.<computed> (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:22875)
    at Object.call (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:36:5137)
    at l.s (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:5145)
    at l.q (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:4647)
    at n.value (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at t.y (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:29:665)
    at t.fire (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:29:882)
    at process.ue (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:25:10030)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[08:38:35] Error [CodeExpectedError]: Could not find pty on pty host
    at D.U (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:32471)
    at D.updateTitle (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:27632)
    at j.<computed> (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:22875)
    at Object.call (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:36:5137)
    at l.s (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:5145)
    at l.q (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:4647)
    at n.value (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at t.y (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:29:665)
    at t.fire (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:29:882)
    at process.ue (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:25:10030)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at D.U (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:32471)
    at D.updateTitle (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:27632)
    at j.<computed> (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:22875)
    at Object.call (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:36:5137)
    at l.s (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:5145)
    at l.q (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:4647)
    at n.value (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at t.y (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:29:665)
    at t.fire (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:29:882)
    at process.ue (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:25:10030)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[08:38:35] Error [CodeExpectedError]: Could not find pty on pty host
    at D.U (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:32471)
    at D.updateTitle (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:27632)
    at j.<computed> (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:22875)
    at Object.call (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:36:5137)
    at l.s (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:5145)
    at l.q (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:4647)
    at n.value (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at t.y (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:29:665)
    at t.fire (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:29:882)
    at process.ue (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:25:10030)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at D.U (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:32471)
    at D.updateTitle (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:27632)
    at j.<computed> (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:22875)
    at Object.call (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:36:5137)
    at l.s (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:5145)
    at l.q (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:4647)
    at n.value (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at t.y (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:29:665)
    at t.fire (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:29:882)
    at process.ue (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:25:10030)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[08:38:36] Error [CodeExpectedError]: Could not find pty on pty host
    at D.U (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:32471)
    at D.updateTitle (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:27632)
    at j.<computed> (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:47:22875)
    at Object.call (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:36:5137)
    at l.s (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:5145)
    at l.q (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:4647)
    at n.value (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at t.y (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:29:665)
    at t.fire (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:29:882)
    at process.ue (/home/pi/.vscode-server/bin/5c3e652f63e798a5ac2f31ffd0d863669328dc4c/out/vs/platform/terminal/node/ptyHostMain.js:25:10030)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[10:46:36] [File Watcher (node.js)] Watcher shutdown because watched path got deleted
[10:59:29] [127.0.0.1][cddf52bf][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[10:59:29] [127.0.0.1][db7a493e][ExtensionHostConnection] <861> Extension Host Process exited with code: 0, signal: null.
Last EH closed, waiting before shutting down
[10:59:29] Last EH closed, waiting before shutting down
